src/components/app.ts

```diff
@@ -0,0 +1,22 @@
+import { createSpan, createDiv } from './helpers/htmlCreator';
+import classes from './styles/collapsed.module.css';
+
+/**
+ * Create widget on the document
+ */
+function createWidget(): void {
+  const container = createDiv(classes.container);
```

Comment:
`createDiv` and `createSpan` now look redundant. 

```suggestion
  const container = make('div', classes.container);
```

Reply:
I slightly disagree, we hide args for function (and make is hidden too). It's more safe + easy to recognise name of function shown meaning of action

Reply:
and you will create separate functions for every HTML tag? They all will be the same, so it is just a code duplication.

>It's more safe

what do you mean?

--

Anyway, it's not a big problem for me, if you want, you can leave them.

Reply:
Because createSpan contains more than make ("span", ...)
`const createSpan = (
  title: string,
  classNames: string | string[]
): HTMLSpanElement => {
  const spanContainer = make('span', classNames);
  const titleValue = document.createTextNode(title);

  spanContainer.appendChild(titleValue);

  return spanContainer;
};`

We use  СreateTextNode  for adding text inside createSpan.
If i remove it, i will doublicate createTextNode twice